api/db/migrate/20100107141738_add_api_key_to_spree_users.rb

```diff
@@ -1,7 +0,0 @@
-class AddApiKeyToSpreeUsers < ActiveRecord::Migration[4.2]
```

Comment:
I think we should deal with these migrations in a separate PR. The weirdness of them came up when I did the roll-up but I didn't want to change any behaviour.

Reply:
Removed this change.

Replaced the db:migrate task with a custom migration task which runs migrations one engine at a time (which simulates running `railties:install:migrations` before `db:migrate`).